src/scraper.py

```diff
@@ -1,27 +1,31 @@
-from bs4 import BeautifulSoup
+from bs4 import BeautifulSoup, element
 import requests
 
 
-def fetch_data_fromXML(todaysDate):
-    r = requests.get('http://www.sksdb.hacettepe.edu.tr/YemekListesi.xml')
+def fetch_data_fromXML(todaysDate: str) -> tuple[list[str], str]:
+    r: requests.Response = requests.get(
+        'http://www.sksdb.hacettepe.edu.tr/YemekListesi.xml')
     r.encoding = 'utf-8'
-    xml_text = r.text
+    xml_text: str = r.text
 
-    soup = BeautifulSoup(xml_text, 'xml')
+    soup: BeautifulSoup = BeautifulSoup(xml_text, 'xml')
 
-    meals = []
-    calorie = 0
+    meals: list[str] = []
+    calorie: str = ''
+
+    day: element.Tag
     for day in soup.select('gun'):
-        date = day.select_one('tarih').text.split()[0]
+        date: str = day.select_one('tarih').text.split()[0]
 
         if date == todaysDate:
-            for meal in day.select('yemek'):
-                meal = meal.text.strip()
+            meal_tag: element.Tag
+            for meal_tag in day.select('yemek'):
+                meal_str: str = meal_tag.text.strip()
 
-                if meal:
-                    meals.append(meal)
+                if meal_str:
+                    meals.append(meal_str)
 
-            calorie = day.select_one('kalori').text
+                calorie = day.select_one('kalori').text
```

Comment:
Oh, I see. I guess it is probably because of my inattention (it is kinda hard for me to focus especially when I am exhausted) thus, I am going to fix it.